src/js/new-visualization-modal.js

```diff
@@ -2,41 +2,34 @@ import * as app from './app';
 
 import dataQuery from '../data/data_query.sql';
 
+let queryResults = null;
 // handle modal submit
 function onNewVisSubmit() {
```

Comment:
add blank space above all functions (obviously, above the comments. Keep the comments touching the function def)

Reply:
It is followed for the most part. Some might have been overlooked. 
Fixed.